tests/integration.rs

```diff
@@ -348,12 +348,7 @@ mod integration_tests {
             },
         )?;
 
-        let expected = r#"Error: Couldn't find trigger executor for local app "spin.toml"
-
-Caused by:
-      no triggers in app
-"#;
-
+        let expected = "Error: No triggers in app\n";
```

Comment:
This error might be a bit too terse. I'm not sure what a better wording would be though...